draft-ietf-quic-transport.md

```diff
@@ -420,8 +361,9 @@ conditions for opening a stream are slightly more complex for a bidirectional
 stream because the opening of either send or receive sides causes the stream
 to open in both directions.
 
-An endpoint can open streams up to its maximum stream limit in any order,
-however endpoints SHOULD open the send side of streams for each type in order.
+An endpoint can open streams up to its maximum stream limit in any order of
+stream ID. However, within each type, endpoints SHOULD open the send side of
+streams in numerically increasing order of stream ID.
```

Comment:
I think that this text was rotten to begin with.  There might be a simpler statement.

"An endpoint MUST open streams of the same type in increasing order of stream ID."